README.md

```diff
@@ -75,9 +75,13 @@
       - [`editField(id[, opts])` : [Field](#field)](#editfieldid-opts--fieldfield)
       - [`deleteField(id)` : void](#deletefieldid--void)
       - [`changeFieldId (currentId, newId)` : void](#changefieldid-currentid-newid--void)
-      - [`changeEditorInterface (fieldId, widgetId[, settings])` : void](#changeeditorinterface-fieldid-widgetid-settings--void)
-      - [`resetEditorInterface (fieldId)` : void](#reseteditorinterface-fieldid--void)
-      - [`copyEditorInterface (sourceFieldId, destinationFieldId)` : void](#copyeditorinterface-sourcefieldid-destinationfieldid--void)
+      - [`changeFieldControl (fieldId, widgetId[, settings, widgetNamespace])` : void](#changefieldcontrol-fieldid-widgetid-settings-widgetnamespace--void)
```

Comment:
Can the signature be `changeFieldControl(fieldId, widgetNamespace, widgetId[, settings])`?

It's more logical (widget lives in a namespace, not the other way around) and also makes it mandatory which is not enforced by the API (it would be breaking change) but in the tooling should be enforced.

Reply:
Well the argument order is based on the old methods to make the sig compatible

Reply:
Yeah but we introduce a new method so it's a chance for us to make things better?

Reply:
changed it